### ISSUE TITLE
implement --synonym argument

### ISSUE THREAD

**Issue by Konfekt** [xueyuanl/cambrinary#6]
Though documented in `--help`, the argument `--synoym` is not recognized, whereas `-s` does not change the output.

**Comment by xueyuanl** [comment]
Hi Konfekt,

Thanks for your use and the feedfack. I have pushed a commit to fix this issue.
test passed by,
```
 cambrinary -s -w small
```
the result is align with the real web page result.

And this fix would probably not push to the PyPI very quick, maybe with the next big release.
So you could just build it locally by,
```
# git clone
# cd into the repo
pip3 install .
```
Done.

Also, I will not close this issue until it comes to the next release.

**Comment by Konfekt** [comment]
Thank you! `pip3 install .` did not work but `cd cambrinary && pip3 install --user --upgrade -r requirements.txt` did. I can confirm `-s` works now.

**Comment by xueyuanl** [comment]
included in next release, v0.1.2.
